packages/antd/src/ModalAnt.tsx

```diff
@@ -1,21 +1,49 @@
-import { Modal as MoxbModal } from '@moxb/moxb';
+import { Action, Modal as MoxbModal } from '@moxb/moxb';
 import { Modal } from 'antd';
 import { ModalFuncProps } from 'antd/lib/modal';
 import { observer } from 'mobx-react';
 import * as React from 'react';
 
 export interface BindModalAntProps<T> extends ModalFuncProps {
     operation: MoxbModal<T>;
+    footer?(actions: Action[]): React.ReactNode;
 }
 
+// same sizes as semantic UI's modals
+const SIZES = {
+    default: '900px', // default size for ant modal: 512px
+    mini: '360px',
+    tiny: '540px',
+    small: '720px',
+    large: '1080px',
+    fullscreen: '95% !important',
+};
+
 @observer
 export class ModalAnt<T> extends React.Component<BindModalAntProps<T>> {
     /*
      * Ant Design has no concept for multiple actions in a modal dialog, it will always be an 'okButton' and
      * a 'cancelButton', so we will warn the user, that not more actions are available.
      * */
     render() {
-        const { operation, children, ...props } = this.props;
+        const { operation, children, footer, width, ...props } = this.props;
```

Comment:
@Mytrill : Wouldn't it be clearer to use an explicit `cancelAction`, `confirmAction` and `footerActions` properties instead of single the `actions` array where you have to check the length of the array of the `actions`

And this way it'd be a bit clearer, what is happening here.